### DISCUSSION TITLE
Wrong type and provider in Request-URL for Curations from ClearyDefined

### DISCUSSION THREAD

**Discussion by DoktorLenz** [oss-review-toolkit/ort#8539] [Q&A] [answered]
Hi there,

i noticed that in the scan step requests to clearlydefined are being made and cached in the ort cache.
But in the scan result no curations from clearlydefined have been applied.
So i checked the cached requests and noticed that the request use a wrong url.
E.g. I use the Package WinUIEx@2.3.4 from the Nuget Package Manager.
The request to clearlydefined used by the scan step is to the following url:
https://api.clearlydefined.io/harvest/git/github/-/WinUIEx/03f6b0992d865d8b796d074d1b4a872a0cdaf79f?form=list
Which returns an empty array.
I wondered why the request uses the git type and github provider instead of using nuget.
So i manually changed the url to
https://api.clearlydefined.io/harvest/nuget/nuget/-/WinUIEx/2.3.4
which returns a propper result with.
Do i need to configure something so that ort uses the right type and provider or is this a bug?

FYI I am using ORT 20.0.0

Thanks in advance!

PS: I just found out that the url with git as type and github as provider works if the namespace would have been set correcty to "dotmorten", which is not the case with ort but manully it works:
https://api.clearlydefined.io/harvest/git/github/dotmorten/winuiex/03f6b0992d865d8b796d074d1b4a872a0cdaf79f?form=list

**Comment by sschuberth** [comment, marked as answer]
> Do i need to configure something so that ort uses the right type and provider or is this a bug?

It's a bug. Something that I've personally been aware of for some time, and I've already [started a branch](https://github.com/oss-review-toolkit/ort/tree/sschuberth/multiple-cd-coordinates) as thought I'd be able to fix it quickly, but it turned out to require some larger refactoring.

So let me [create an issue](https://github.com/oss-review-toolkit/ort/issues/8540) from this discussion to track it.

**Reply by DoktorLenz** [reply]
All right, thanks for the quick reply.
